user/migrations/0005_alter_user_managers.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.1 on 2024-12-17 06:12
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("user", "0004_alter_user_managers"),
+    ]
+
+    operations = [
+        migrations.AlterModelManagers(
+            name="user",
+            managers=[],
+        ),
```

Comment:
This migration removes all managers from the `User` model. Ensure this is intentional, as it will prevent standard query operations on the `User` model. If this is not the desired behavior, consider specifying a default manager.